odl/trafos/wavelet.py

```diff
@@ -286,6 +286,28 @@ def _call(self, x):
         else:
             raise RuntimeError("bad `impl` '{}'".format(self.impl))
 
+    def scales(self):
```

Comment:
A doctest would indeed be nice, it took me a minute to understand what this function does. And yes, it would make sense to move it into the base class. The only "tricky" part is to determine the wavelet space, everything else is just the same code.